app/components/MessageList.js

```diff
@@ -1,30 +1,44 @@
 import React, { PureComponent } from 'react'
 
-import { FlatList, View, Text } from 'react-native'
+import { FlatList, View, Text,StyleSheet,TextInput } from 'react-native'
 import PropTypes from 'prop-types'
+import ChatView from './ChatView'
+import { addMessage,sendMessage } from '../redux/chat'
+import users from '../redux/users';
 
 export default class MessageList extends PureComponent {
+ 
   _keyExtractor = (item, index) => item._id
 
-  _renderItem = ({item}) => (
-    <View>
-      <Text>
-        {item.text}
-      </Text>
+  _renderItem = ({item,user,friend}) => (
+    <View >
```

Comment:
Do we need <View> outside?

Reply:
we don't need this outside .It will when we just pass in the component